blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""This module contains models for the blog app."""
```

Comment:
Could you decide whether it's "app", "blog app" or "Blog app"? I saw different inconsistent spellings of a seemingly the same thing.

Reply:
Thank you. It's make sense.

Reply:
Marking as resolved.